vimrc

```diff
@@ -221,3 +221,4 @@ else
   let &t_EI = "\<Esc>]50;CursorShape=0\x7"
 end
 
+set shell=/bin/sh
```

Comment:
does this need to be the same as the tmux shell?  I believe our iterm/tmux config uses zsh